cli/src/config/templates.toml

```diff
@@ -33,8 +33,8 @@ separate(" ",
 
 config_list = '''
 if(overridden,
-  label("overridden", indent("# ", name ++ " = " ++ value)),
-  name ++ " = " ++ value,
+  label("overridden", indent("# ", name ++ " = " ++ value ++ " " ++ origin)),
+  name ++ " = " ++ value ++ " " ++ origin,
 ) ++ "\n"
```

Comment:
The default template output is a valid TOML document. Maybe we can use comment syntax?

It might be also make sense to add `builtin_config_list_detailed` or something. We wouldn't want to see the source type/path by default.

Reply:
> The default template output is a valid TOML document. Maybe we can use comment syntax?

Ah good point